php/libraries/rest_api/endpoints/Candidates.class.inc

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace LORIS\RestAPI;
+
+use \LORIS\Core\LORIS_DAO_Factory;
+
+class Candidates extends Endpoint
+{
+    private $_candidate;
+    private $_allowedMethods = array('GET','POST', 'PUT', 'DELETE');
```

Comment:
As per our specifications `DELETE` is not implemented in any of our API's endpoints